doc/application/index.rst

```diff
@@ -253,18 +253,17 @@ section describes the most important ones that every Zephyr developer
 should know about.
 
 .. note::
-   All variables listed in this section can be supplied to the build system
-   in 3 ways (in order of precedence):
+
+   The variables :makevar:`BOARD`, :makevar:`CONF_FILE`, and
+   :makevar:`DTC_OVERLAY_FILE` can be supplied to the build system in
+   3 ways (in order of precedence):
```

Comment:
Since you are editing this, can you please clarify if the list of files for the `_FILE` variables need to be space-separated or semicolon-separated?

Reply:
Sure thing